### ISSUE TITLE
`disableFeature` test helper not available in current release

### ISSUE THREAD

**Issue by nwhittaker** [kategengler/ember-feature-flags#107]
The README indicates a `disableFeature` test helper is available[1]. However, the `6.0.0` release does not include this test helper which was added later via #97.

---
[1] https://github.com/kategengler/ember-feature-flags#enablefeature--disablefeature

**Comment by kategengler** [comment]
The README will always document the main branch, if you want to see documentation for old version you can view the README for the tag https://github.com/kategengler/ember-feature-flags/tree/v6.0.0

Are you asking for a release?

**Comment by nwhittaker** [comment]
>Are you asking for a release?

I was hoping for that outcome, yes. Is that doable?

**Comment by timiyay** [comment]
I've hit this bug today, when attempting to use `disableFeature` and getting back:

```
TypeError: (0, _testSupport.disableFeature) is not a function
```

If a release of this package reinstates this function, is it possible to get a 6.0.1 release to fix this issue?

**Comment by sepuckett86** [comment]
I agree with the above comments -- I'd be interested in using disableFeature when a 6.0.1 release is possible.

**Comment by SergeAstapov** [comment]
+1 for the release.

@kategengler is there anything we could do to help with releasing new version?

**Comment by Techn1x** [comment]
I contributed this feature back in 2020
https://github.com/kategengler/ember-feature-flags/pull/97

I had already been wanting `disableFeature` for a long time before that, that's why I contributed it.

I feel my contribution has actually had a negative effect, as people attempt to use it based on its presence in the README, and then they waste a bunch of time trying to get it to work (or even authoring & structuring test code around the assumption that the test helper exists), before realising that this addon hasn't cut a release in 4 years.

I would appreciate a release being cut with my contribution, though I understand that this addon may not be in a state where doing a patch a release is straightforward.

Should this addon be donated to the adopted addons project? It seems like a strong candidate.

**Comment by kategengler** [comment]
This was released in 6.1.0. Sorry for the long delay.
